packages/resizin/src/upload/uploadFactory.ts

```diff
@@ -0,0 +1,20 @@
+import upload, { FileType } from './upload';
+import { DEFAULT_API_URL } from '../constants';
+
+interface ClientOptions {
+    serverUrl?: string;
+    apiKey: string;
+    fileType?: FileType;
+}
+
+const uploadFactory = (options: ClientOptions = {} as ClientOptions) => (
```

Comment:
ClientOptions = {} as ClientOptions? Is this correct?

Reply:
Syntax is correct, but maybe it could be written better way, I'll look at it.